tests/thread_stats.c

```diff
@@ -0,0 +1,80 @@
+#include <thread.h>
+#include <ktest.h>
+#include <klog.h>
+#include <sched.h>
+#include <runq.h>
+
+#define THREADS_NUMBER 10
+#define TEST_TIME 500
```

Comment:
500 µs? 500ms? 500s? Please use `timeval_t` values to explicitely express time unit:
```c
const timeval_t test_time = TIMEVAL(0.5);
```